package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "jquery-ajax-unobtrusive",
+  "version": "3.2.5",
+  "description": "Add-on to jQuery Ajax to enable unobtrusive options in data-* attributes",
+  "main": "jquery.unobtrusive-ajax.js",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/aspnet/jquery-ajax-unobtrusive"
+  },
+  "keywords": [
+    "unobtrusive",
+    "ajax",
+    "mvc",
+    "asp.net",
+    "jquery"
+  ],
+  "author": "Microsoft",
+  "license": "https://www.microsoft.com/web/webpi/eula/aspnetcomponent_rtw_enu.htm",
```

Comment:
Hmm, is this the EULA we have on other JS files or something? I think we want to usehttps://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm

E.g. see what we do in Coherence-Signed for all the NUPKGs: https://github.com/aspnet/Coherence-Signed/blob/dev/dnx.msbuild#L682

Do we need to go update other JS files?